actions/qemuimg2disk/v1/Dockerfile

```diff
@@ -0,0 +1,67 @@
+FROM alpine:3.13.5 as builder
+ENV QEMU_NAME="qemu"
+ENV QEMU_REV="6.0.0-rc3"
+ENV QEMU_SRC_BASENAME="${QEMU_NAME}-${QEMU_REV}"
+ENV QEMU_SRC_URL="https://download.qemu.org/${QEMU_SRC_BASENAME}.tar.xz"
+
+RUN apk update
+RUN apk upgrade
+
+# required by qemu
+RUN apk add\
+ make\
+ samurai\
+ perl\
+ python3\
+ gcc\
+ libc-dev\
+ pkgconf\
+ linux-headers\
+ glib-dev glib-static\
+ zlib-dev zlib-static
+
+# additional
+RUN apk add bash xz git patch parted
+
+# static libraries for qemu-img
+RUN apk add curl-dev curl-static nettle-dev nettle-static brotli-dev brotli-static nghttp2-dev nghttp2-static openssl-libs-static
```

Comment:
should probably install all the packages at once, to be more "atomic". I think you could do `--virtual` packages to group them similar to the commented info if you want.

Reply:
Yeah, that's leftover from development when I was doing iterative work and I wanted to take advantage of docker layer caching. None of that persists in the resulting image as I pull out only needed binaries into a clean container at the end of the build.  Cleaned up a bit all the same.